webdriver-spec.html

```diff
@@ -5068,6 +5068,12 @@
  In case the <a>element</a> is not <a>keyboard-interactable</a>,
  an <a>element not interactable</a> <a>error</a> is returned.
 
+<p>Calling this command effectively clears the <a>current
+ session</a>'s <a>input state</a>. The <a>key input state</a> used for
```

Comment:
This isn’t what I meant. We shouldn’t clear any input state when running this command.

What I meant is that each invocation of Element Send Keys uses a new temporary, virtual keyboard device and consequently its state is cleared when the command is done.